frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -19,5 +53,158 @@ export const personsModalLogic = kea<personsModalLogicType>({
                 setCohortModalVisible: (_, { visible }) => visible,
             },
         ],
+        people: [
+            null as TrendPeople | null,
+            {
+                setFilters: () => null,
+                setPeople: (_, { people }) => people,
+                setFirstLoadedPeople: (_, { firstLoadedPeople }) => firstLoadedPeople,
+            },
+        ],
+        peopleLoading: [
+            false,
+            {
+                setPeopleLoading: (_, { loading }) => loading,
+            },
+        ],
+        firstLoadedPeople: [
+            null as TrendPeople | null,
+            {
+                saveFirstLoadedPeople: (_, { people }) => people,
+            },
+        ],
+        loadingMorePeople: [
+            false,
+            {
+                setLoadingMorePeople: (_, { status }) => status,
+            },
+        ],
+        showingPeople: [
+            false,
+            {
+                loadPeople: () => true,
+                setShowingPeople: ({}, { isShowing }) => isShowing,
+            },
+        ],
+    }),
+    listeners: ({ actions, values }) => ({
+        refreshCohort: () => {
+            cohortLogic({
+                cohort: {
+                    id: 'new',
+                    groups: [],
+                },
+            }).actions.setCohort({
+                id: 'new',
+                groups: [],
+            })
```

Comment:
Could there be a case ever where this logic could already exist and be used for something else? E.g. the modal is used on `/cohorts/new`? Perhaps a `"new2"` (or better) id could be used to separate this usage?